pages/opam-packaging.html

```diff
@@ -1,86 +1,191 @@
 <#def TITLE>Creating and Submitting a Package</#def>
 <#include "incl/header.html">
-<p>This page contains instructions covering the case of a simple Coq package
-called <code>foo</code> with version <code>1.0.0</code>.
-The full documentation on OPAM packages can be found on the <a href="http://opam.ocaml.org/">OPAM web site</a>.</p>
+<p>This page contains instructions to submit a simple example Coq package
+called <code>foo</code> at version <code>1.0.0</code>.
+The full documentation on Opam packages can be found on the
+<a href="http://opam.ocaml.org/doc/Packaging.html">Opam web site</a>.</p>
 
-<h2>Creating a package</h2>
+<h2>Package layout</h2>
 
-<p>First, go to the github page of <a href="https://github.com/coq/opam-coq-archive">opam-coq-archive</a> and <a href="https://help.github.com/articles/fork-a-repo/">make a fork</a>.</p>
+<p>For simplicity, we will assume that the Coq project you want to package
+contains a <code>Makefile</code> providing the following commands:
+</p>
 
-<p>The repository is now available in your personal github space.
-Clone it locally by typing the following, where <code>user</code> is your github user name.</p>
+<pre><code>make          # Build the package
+make install  # Install the package
+</code></pre>
 
-<pre><code>git clone git@github.com:user/opam-coq-archive.git
-cd opam-coq-archive
+<details>
+  <summary>Click here to see a minimal example <code>Makefile</code>.
+  </summary>
+<pre><code>build: Makefile.coq
+	$(MAKE) -f Makefile.coq
+
+install: Makefile.coq
+	$(MAKE) -f Makefile.coq install
+
+Makefile.coq: _CoqProject
+	coq_makefile _CoqProject -o $@
```

Comment:
isn't it `coq_makefile -f _CoqProject -o ...`

Reply:
Uh oh, you're right!